test/test_setup.jl

```diff
@@ -31,7 +31,7 @@
         rc = RemoteChannel(()->Channel(2*length(PH.scenarios(st))), myid())
 
         PH._build_subproblems(rc, wr, PH.scenarios(st), st,
-                              create_model, (), NamedTuple())
+                              create_model, (), NamedTuple(), SubproblemCallback[])
```

Comment:
More unusual formatting.

Reply:
I'm not sure what you mean by unusual formatting here - it's the same as before but with the additional `SubproblemCallback[]`.

Reply:
Nevermind. Looks like github displayed it in an unusual way.  Sorry about that.